### ISSUE TITLE
If Jobs are not enabled then the /job endpoint should be disabled and removed from Swagger

### ISSUE THREAD

**Issue by p013570** [gchq/Gaffer#1421]
If jobs are not enabled then the JobService classes should not be added in the ApplicationConfig classes.

You may need to add a method to Graph/Store 'areJobsEnabled' to return true/false based on if the jobTracker has been configured.

**Comment by p013570** [comment]
Should be done after #1513.

**Comment by p013570** [comment]
Merged into develop.
